imap_processing/ena_maps/ena_maps.py

```diff
@@ -380,62 +401,21 @@ class AbstractSkyMap(ABC):
 
     @abstractmethod
     def __init__(self) -> None:
-        pass
+        self.tiling_type: SkyTilingType
+        self.sky_grid: spatial_utils.AzElSkyGrid
+        self.num_points: int
+        self.data_dict: dict[str, NDArray]
 
-    def __repr__(self) -> str:
+    def copy(self) -> AbstractSkyMap:
```

Comment:
Are we using this anywhere / do we need it? A user could do the deepcopy themselves. Also, xarray has a `deep=True` keyword to choose whether to deepcopy or not, so we should maybe use that syntax if we are going to add it: https://docs.xarray.dev/en/latest/generated/xarray.DataArray.copy.html

But, my preference is generally to leave this up to end-users / ourselves in the instruments to do a deepcopy if necessary. (Might be worth leaving this for another PR with just this feature added if so)

Reply:
We can get away without it. I had been planning on making copies in some tests, but if necessary, we can just apply deepcopy when necessary. 

The need for copying might come back at some point, if we, say want to make a map at a certain resolution and then downsample it while keeping the original. I'll cut it for now!